lib/alpha_beta.py

```diff
@@ -29,8 +29,19 @@ def timeout_test(start_time: float):
     return delta_time > MAX_RESPONSE_TIME
 
 
-def evaluate(state: ndarray, game_type: str):
-    return absolute_heuristic(state, game_type)
+def evaluate(state: ndarray, game_type: str, heuristic_played: str):
+    if heuristic_played == "absolute":
```

Comment:
Could we use dictionary mapping instead? 

Something like:
```
{
  "absolute": absolute_heuristic,
  "heuristic_2": lambda state, game_type: heuristic_2(
     state=state,
     species_played=game_type,
     population_weight=10000,
     absolute_win_weight=1,
     random_fight_weight=1,
     human_win_weight=1,
     winning_weight=100000,
    )
}
```

Reply:
We can, but why do you think it's better ?

Reply:
For readability purposes and also to make sure typos won't lead to unexpected bugs. It makes the whole thing easier to scale up.